sretoolbox/container/image.py

```diff
@@ -178,8 +195,13 @@ def manifest(self):
         Property to cache the manifest returned but get_manifest()
         """
         if self._cache_manifest is None:
-            manifest = self._get_manifest()
-            self._cache_manifest = manifest.json()
+            try:
+                manifest = self.response_cache[self.digest]
```

Comment:
Both digests and etags as keys to the same cache? Is this intentional or a leftover?
Isn't this cached already at the request level?

Reply:
It is intentional - it may be already cached at the request level, if the registry did return an etag. Or it may need caching here if the registry doesn't.
If there is an etag, it most likely will be identical to the digest - in the rare registry that doesn't, oh, well, we've wasted 10KB a few times.

Reply:
got it. tks for clarifying.